src/internal/Transition.js

```diff
@@ -1,7 +1,5 @@
 // @flow weak
-
```

Comment:
@mbrookes @oliviertassinari - I have no preference, other than to say that I don't want to make this a constant PR hassle.  If this is to be something we want, it should be enforced by eslint.